integr8ly/cloud-resource-operator
# Create version 0.22.2

## Overview

Bump CRO version to v0.22.2
https://issues.redhat.com/browse/INTLY-10249

## Verification
- Run end to end tests
- Install operator and confirm version 0.22.2
